### DISCUSSION TITLE
How do I use .py file to identify the formula

### DISCUSSION THREAD

**Discussion by slimsu** [lukas-blecher/LaTeX-OCR#244] [Q&A]
Hello, instead of typing latexocr on the command line to open the window and typing pix2tex plus image path to get the result, which .py file should I run to process the image I want to put in the prediction.
I want to use .py file to run the program, which can process more than one image in order. I ran gui.py and cli.py, but there was no error and no reaction. I don't know where to put the image I want to predict, could you please answer it for me, thank you

**Comment by lukas-blecher** [comment]
You can write your own script, see #238

**Reply by slimsu** [reply]
Ok, thank you very much
